audit/format.go

```diff
@@ -557,3 +578,16 @@ func doElideListResponseData(data map[string]interface{}) {
 		}
 	}
 }
+
+// getMountClass returns the mount class based the mount accessor of a logical.Request.
+func getMountClass(req *logical.Request) string {
+	if req.MountAccessor == "" || strings.HasPrefix(req.Path, "sys/") {
```

Comment:
Does this work if the request is under a namespace?

Reply:
I believe this works if the request is under a namespace since `req.Path` has the namespace trimmed further up in the call stack. For example, in `buildLogicalRequestNoAuth()` that constructs `logical.Request` , `logical.Request.Path` is made from namespace-trimmed `http.Request.URL.Path` https://github.com/hashicorp/vault/blob/main/http/logical.go#L53.

Reply:
I decided to use add `MountClass()` to `MountEntry` instead of using `logical.Request.Path` as `MountEntry.Path` doesn't have a namespace.

```
// APIPath returns the full API Path for the given mount entry
func (e *MountEntry) APIPath() string {
	path := e.Path
	if e.Table == credentialTableType {
		path = credentialRoutePrefix + path
	}
	return e.namespace.Path + path
}

// APIPathNoNamespace returns the API Path without the namespace for the given mount entry
func (e *MountEntry) APIPathNoNamespace() string {
	path := e.Path
	if e.Table == credentialTableType {
		path = credentialRoutePrefix + path
	}
	return path
}
```